src/pages/gen2/reference/cli-commands/index.mdx

```diff
@@ -215,9 +215,82 @@ npx amplify generate config --stack amplify-nextamplifygen2-josef-sandbox-ca85e1
+Sometimes you want to test your latest local changes with the backend of another deployed branch. If you want to generate the GraphQL client code file(s) for the latest deployment of another branch, then you can run the following command:
+
+```bash 
+npx amplify generate graphql-client-code --branch staging
+```
+### Generate codegen for CDK app using a joint "AmplifyBackendStack" construct
+Assume you have deployed your Amplify project with the CDK construct. You will need to remember your app's project name (designated as the second parameter in your CDK construct) and stack name (designated as part of your `npx cdk deploy` context)
+
+// CDK code in stack.ts
+```ts
+import { Construct } from 'constructs';
+import { App, Backend } from 'aws-cdk-lib/aws-amplify';
+
+export class HelloCdkStack extends cdk.Stack {
+  constructor(scope: cdk.Construct, id: string, props?: cdk.StackProps) {
+    super(scope, id, props);
+
+    new Backend(this, "my-amplify-backend", { /* ... */ });
+  }
+}
+```
+// Deployment command for CDK project
+```bash
+npx cdk deploy
+```
+Run Amplify codegen command to generate GraphQL codegen:
+```bash
+npx amplify generate graphql-client-code --stack="my-amplify-backend" --platform ts --out ./src
+```   
+### Generate codegen in specific language and format
+
+```bash
+npx amplify generate graphql-client-code --format modelgen --type-target angular
+```
+### Supported GraphQL client code combinations:
+
+| Format | Platform | Codegen command in Amplify CLI | Command in Amplify Gen2 | Default generated file/path |
+| -------| -------- | ------------------------------ | ----------------------- | --------------------------- |
+| Introspection schema | Amplify Javascript | N/A | `npx amplify generate graphql-client-code --format introspection` | `<path_to_app>/` |
+| GraphQL codegen | Amplify Javascript | `amplify codegen` | `npx amplify generate graphql-client-code -format graphql-codegen --statement-target javascript --out <path_to_app>/src/graphql/` | `<path_to_app>/src/graphql/` |
+| Modelgen | Amplify Javascript | `amplify codegen model` | `npx amplify generate graphql-client-code --format modelgen --model-target javascript --out <path_to_app>/src/models/` | `<path_to_app>/src/models/` |
+| Modelgen | Amplify Android | `amplify codegen model` | `npx amplify generate graphql-client-code --format modelgen --model-target java --out <path_to_app/src/main/java/>` | `<path_to_app>/src/main/java/com/amplifyframework/datastore/generated/model` |
+| Modelgen | Amplify Swift | `amplify codegen model` | `npx amplify generate graphql-client-code --format modelgen --model-target swift --out <path_to_swift_project>/AmplifyModels` | `<path_to_swift_project>/AmplifyModels` |
+| Modelgen | Amplify Flutter | `amplify codegen model` | `npx amplify generate graphql-client-code --format modelgen --model-target dart --out <path_to_flutter_project>/AmplifyModels` | `<path_to_flutter_project>/AmplifyModels` |
```

Comment:
does this render okay? the commands are a bit lengthy for a column

Reply:
yes.